examples/llm/components/prefill_worker.py

```diff
@@ -64,6 +64,12 @@ def __init__(self):
             print("Prefill must be done eagerly, setting to True")
             self.engine_args.enforce_eager = True
 
+        if self.engine_args.enable_prefix_caching is not False:
```

Comment:
If prefix caching is not supported in PrefillWorker what gain are we expecting in passing the cached KV?